src/index.scss

```diff
@@ -5,7 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  background: url('./bck.jpg');
+  background: url('./bck.jpg') no-repeat;
```

Comment:
setting an image here works, but it is not ideal.
why? because it's static. what if later on down the road, we want to be able to dynamically change the background? we wouldn't be able to do so unless we change the code and restart the app.
if we set the image on the `Component` level, we can easily change the background during runtime if we want.

**NOTE**: dont' make the change on this branch. any component changes we make should just come after we import the UI library.

Reply:
I did not change anything here -- this is good to know. Thank you! Should I import the library in another ticket?

Reply:
yeah. we can have a separate branch after this is merged.
that imports the material-UI and update the components to use it